website/templates/errors/401.html

```diff
@@ -20,8 +20,4 @@
 
 {% block error_message %}Error 401: Unauthorized{% endblock %}
 
-{% block error_caption %}New app, who dis?{% endblock %}
-
-{% block error_tips %}
-If you're logged out, try logging in to the application.
```

Comment:
Did we mean to remove the "Try logging in" notice?

Reply:
Thought it was more self-explanatory, but will add back in.